linux_os/guide/system/software/system-tools/package_python3-abrt-addon_removed/rule.yml

```diff
@@ -0,0 +1,29 @@
+documentation_complete: true
+
+prodtype: fedora,rhel8
+
+title: 'Uninstall python3-abrt-addon Package'
+
+description: |-
+    {{{ describe_package_remove(package="python3-abrt-addon") }}}
+
+rationale: |-
+    <tt>python3-abrt-addon</tt> contains python hook and python analyzer
+    plugin for handling uncaught exceptions in python programs.
+
+severity: low
+
+identifiers:
+    cce@rhel8: CCE-82923-4
```

Comment:
I have reused the CCE from RHEL-8, should I actually get a new one?

Reply:
I think that yes, you should, it's a different package, therefore a different configuration setting.

Reply:
New CCE added.